ecospheres_migrator/templates/fragments/transform_job_status.html.j2

```diff
@@ -57,6 +58,60 @@
       </div>
     </section>
     {% endif %}
+    {% if skipped %}
+    <section class="fr-accordion">
+      <h3 class="fr-accordion__title">
+        <button class="fr-accordion__btn"
+                aria-expanded="true"
+                aria-controls="accordion-skipped">Transformations ignorées ({{ skipped|length }})</button>
+      </h3>
+      <div class="fr-collapse" id="accordion-skipped">
+        <div class="fr-table fr-mb-1w" id="table-md-component">
+          <div class="fr-table__wrapper">
+            <div class="fr-table__container">
+              <div class="fr-table__content">
+                <table id="table-md">
+                  <thead>
+                    <tr>
+                      <th scope="col">Statut</th>
+                      <th scope="col">Fiche originale</th>
+                      <th scope="col">XML original</th>
+                      <th scope="col">Raison</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {% for record in skipped %}
+                      <tr id="table-md-row-key-{{ loop.index }}"
+                          data-row-key="{{ loop.index }}">
+                        <td>⏭️</td>
+                        <td>
+                          <a href="{{ url }}/fre/catalog.search#/metadata/{{ record.uuid }}"
+                            target="_blank"
+                            rel="noopener">{{ record.uuid }}</a>
+                        </td>
+                        <td>
+                          <a href="{{ url_for('transform_original', job_id=job.id, uuid=record.uuid) }}"
+                            target="_blank"
+                            rel="noopener">Voir le XML</a>
+                        </td>
+                        <td>
+                          {% if record.reason.name == "NO_CHANGES" %}
```

Comment:
Seems like we could avoid the `if` here, either with more explicit strings in the `(Str)Enum`, or some other lookup table? I can already see that `if` grow with each new reason we add...

Reply:
Explicit strings in `Str(Enum)` are a problem because they're pickled in the job and won't match from one job to another if we change the message. I'll use another mapping for this.